R/utils.R

```diff
@@ -93,3 +93,22 @@ check_rel_susceptibility <- function(rel_susceptibility) {
       "increasing"))
   }
 }
+
+build_waning_rate <- function(waning_rate) {
+  if (length(waning_rate) == 0) {
+    stop("At least one value required for 'rel_susceptibility'")
+  }
+  N_age <- get_n_groups()
```

Comment:
the N_age / n_groups split here is jarring

Reply:
Might be worth finally correcting this so that `N_groups` replaces `N_age` where really we mean `N_age + 2`?

Reply:
suggest doing this in separate PR. At the moment have just removed "N_age" from this specific function